.drone.star

```diff
@@ -855,6 +855,94 @@ def buildRelease(ctx):
 		}
 	]
 
+def website(ctx):
+  return [
+	{
+		'kind': 'pipeline',
+		'type': 'docker',
+		'name': 'website',
+		'platform': {
+			'os': 'linux',
+			'arch': 'amd64',
+		},
+		'steps': [
+			{
+				'name': 'prepare',
+				'image': 'owncloudci/alpine:latest',
+				'commands': [
+				'	make docs-copy'
+				],
+			},
+			{
+				'name': 'test',
+				'image': 'webhippie/hugo:latest',
+				'commands': [
+					'cd hugo',
+				'	hugo',
+				],
+			},
+			{
+				'name': 'list',
+				'image': 'owncloudci/alpine:latest',
+				'commands': [
+					'tree hugo/public',
+				],
+			},
+			{
+				'name': 'publish',
+				'image': 'plugins/gh-pages:1',
+				'pull': 'always',
+				'settings': {
+					'username': {
+						'from_secret': 'github_username',
+					},
+					'password': {
+						'from_secret': 'github_token',
+					},
+					'pages_directory': 'docs/',
+					'target_branch': 'docs',
+				},
+				'when': {
+					'ref': {
+						'exclude': [
+						'refs/pull/**',
+						],
+					},
+				},
+			},
+			{
+				'name': 'downstream',
+				'image': 'plugins/downstream',
+				'settings': {
+					'server': 'https://cloud.drone.io/',
```

Comment:
We don't use our drone instance for this?

Reply:
This is the central docs repo.